docs/Makefile

```diff
@@ -52,3 +66,109 @@ docs-images-to-webp: docs-image
 			content \
 				-regex ".*\.\(png\|jpg\|jpeg\)" \
 				-exec sh -c 'cwebp -preset drawing -m 6 -o $$(echo {} | cut -f-1 -d.).webp {} && rm -rf {}' {} \;
+
+# The command that updates flags in the documentation using the actual binaries compiled
+# from the latest enterprise-single-node and enterprise-cluster branches (hardcoded for now).
+# The command also normalizes the output a bit.
+#
+# It can be run from any branch.
+# All work happens inside temporary directories under /tmp.
+# The script checks out the required branch, builds the binaries, and updates the documentation.
+# The current Git repository is not touched.
+docs-update-flags:
+	# Note for MacOS users:
```

Comment:
can we add a separate flag `--docs-help` or something like this, that formats output in expected manner instead of using bash to cover each case, which anyway should be maintained, while custom command line flag logic can be covered by tests